ESLintエラー

# ESLintエラー

## エラー内容
```bash
npm run lint

> book-manage-service@1.0.0 lint
> eslint ./src/index.ts src/**/*.ts


/root/repositories/github/pxr-book-manage-service/src/tests/04-01.BookCooperate.spec.ts
  553:78  error  Block must not be padded by blank lines  padded-blocks

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

--- a/src/tests/04-01.BookCooperate.spec.ts
+++ b/src/tests/04-01.BookCooperate.spec.ts
@@ -551,7 +551,6 @@ describe('book-mange API', () => {
             // レスポンスチェック
             expect(response.status).toBe(400);
             expect(response.body.message).toBe(Message.EMPTY_REGION_AND_APP);
-
         });
         test('正常：IDサービスを使用しない', async () => {
             _catalogServer = new StubCatalogServerNoIdService(3001, 1000374, 200);

